packages/moonstone/Notification/Notification.js

```diff
@@ -135,13 +135,8 @@ const NotificationBase = kind({
 	},
 
 	computed: {
-		className: ({className, buttons, styler}) => {
-			if (buttons && buttons.length > 3) {
-				return styler.append({wide: true});
-			} else {
-				return className;
-			}
-		},
+		className: ({buttons, styler}) =>
+			styler.append({wide: (buttons && buttons.filter(Boolean).length > 2)}),
```

Comment:
This is a good structural improvement. I did notice that our original logic was wrong in that it didn't correctly check for a single button. In that case, `length` would be undefined since it doesn't exist on a React Element object.

In the updated code, I believe this will throw an error since we'd be invoking an undefined method. Using `React.Children.toArray(buttons)` will insulate us from managing that condition since it returns an array for a single child.

Stylistically, I'd suggest moving `styler` to the previous line and line break the object which is our typical style and is a bit more friendly to diffs.